reconcile/utils/terrascript_aws_client.py

```diff
@@ -3242,11 +3278,11 @@ def populate_tf_resource_ecr(self, spec):
         # https://www.terraform.io/docs/providers/aws/r/iam_access_key.html
 
         # iam user for repository
-        values = {}
-        values["name"] = identifier
-        values["tags"] = common_values["tags"]
-        values["depends_on"] = self.get_dependencies([ecr_tf_resource])
-        user_tf_resource = aws_iam_user(identifier, **values)
+        values_iam_user: dict[str, Any] = {}
+        values_iam_user["name"] = identifier
+        values_iam_user["tags"] = common_values["tags"]
+        values_iam_user["depends_on"] = self.get_dependencies([ecr_tf_resource])
```

Comment:
can just create it once instead of keep mutating

```python
values_iam_user = {
  "name": identifier,
  "tags": common_values["tags"],
  "depends_on": self.get_dependencies([ecr_tf_resource]),
}
```

can apply to multiple places in this file.